scalafix-core/src/main/scala-2.12/scalafix/util/metaconfig/generic/package.scala

```diff
@@ -0,0 +1,21 @@
+package metaconfig
+
+import scalafix.internal.util.MetaconfigCompatMacros
+
+import scala.language.experimental.macros
+
+// shadows https://github.com/scalameta/metaconfig/blob/v0.10.0/metaconfig-core/shared/src/main/scala-2/metaconfig/generic/package.scala
```

Comment:
since https://github.com/coursier/coursier/pull/1546, scalafix-core should be before metaconfig-core